src/main/java/com/uid2/shared/store/parser/ParsingResult.java

```diff
@@ -0,0 +1,19 @@
+package com.uid2.shared.store.parser;
+
+public class ParsingResult<T> {
+    private final T data;
+    private final Integer count;
```

Comment:
Do we need to keep track of the `count`?

Reply:
Yes since all the store methods return the count. Not sure why is that useful though - possibly can get rid of it if nobody needs it - hard to tell with changes in Shared though.